src/test/java/org/opensearch/security/AggregationTests.java

```diff
@@ -42,9 +42,8 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import com.amazon.opendistroforelasticsearch.security.test.SingleClusterTest;
-import com.amazon.opendistroforelasticsearch.security.test.helper.rest.RestHelper;
-import com.amazon.opendistroforelasticsearch.security.test.helper.rest.RestHelper.HttpResponse;
```

Comment:
This should be reverted. There are many other files that have that removes
`import com.amazon.opendistroforelasticsearch.security.test.helper.rest.RestHelper.HttpResponse;`
Should add this back.

Reply:
Done

Reply:
Done.